doc/Support/Configuration.md

```diff
@@ -650,16 +650,34 @@ you don't need to configure full location within snmp.
 
 # Interfaces to be ignored
 
+Interfaces can automatically be ignored during discovery by adding bad_if\* 
+entries in either the OS specific definition (includes/definitions/_specific_os_.yaml) 
```

Comment:
This info is good to include, but we should not suggest users edit yaml files.  This has a high chance to block updates.

FYI yaml can be overriden in config.php.

Reply:
Sweet, alright I see that now under [here](https://docs.librenms.org/Developing/os/Settings/#user-override-in-configphp). I'll modify my documentation updates.

Reply:
I have updated the documentation portion of this pull-request.